htdocs/core/class/html.form.class.php

```diff
@@ -1962,7 +1962,7 @@ public function select_dolusers($selected = '', $htmlname = 'userid', $show_empt
 					if (!is_numeric($show_empty)) {
 						$textforempty = $show_empty;
 					}
-					$out .= '<option class="optiongrey" value="'.($show_empty < 0 ? $show_empty : -1).'"'.((empty($selected) || in_array(-1, $selected)) ? ' selected' : '').'>'.$textforempty.'</option>'."\n";
+					$out .= '<option class="optiongrey" value="'.($show_empty <= 0 ? $show_empty : -1).'"'.((empty($selected) || in_array(-1, $selected)) ? ' selected' : '').'>'.$textforempty.'</option>'."\n";
```

Comment:
By convention (in Dolibarr) on list that list available id, when we want an empty value, we use the value -1 as the empty value. This is because sometimes, we have 0 as possible value for id (even if this is not the case for list of users).

Reply:
> By convention (in Dolibarr) on list that list available id, when we want an empty value, we use the value -1 as the empty value. This is because sometimes, we have 0 as possible value for id (even if this is not the case for list of users).

Ok @eldy i search a solution to display select wording in empty value for look feel and user frendly